server/src/utils/hashingUtil.js

```diff
@@ -0,0 +1,15 @@
+const bcrypt = require('bcrypt');
+
+const saltCount = process.env.SaltRound;
+
+const HashPass = async(password)=>{
```

Comment:
`camelCase` function names. `PascalCase` is to be used for constructor functions.